kytos/core/switch.py

```diff
@@ -207,6 +226,23 @@ def __init__(self, address, port, socket, switch=None):
         self.port = port
         self.socket = socket
         self.switch = switch
+        self._state = None
+        self.STATES = Connection_states
```

Comment:
I don't see it as a definition, but simply as a shortcut so that when we receive the event as an arg on a handler, we can access the possible states without having to import something new... I'll remove it if you guys want.

About the setter, I thought it was a good way to provide the logging.